tests/order_modifiers_integration/Admin/Editor_Test.php

```diff
@@ -171,13 +171,18 @@ function () {
 	 * @dataProvider asset_data_provider
 	 */
 	public function it_should_locate_assets_where_expected( $slug, $path ) {
+		$this->set_fn_return( 'tec_using_classy_editor', false );
```

Comment:
You can use the `tec_using_classy_editor` filter. Where possible, use a filter over a hard mock.